File: app/src/main/java/com/wozart/aura/data/model/SharedAccess.java
package com.wozart.aura.data.model;

/**
 * Created by wozart on 20/02/18.
 */

public class SharedAccess {
    private String access = "false";
    private String home;
    private String userId;
    private String name;

    public SharedAccess(String access, String home, String userId, String name){
        this.access = access;
        this.home = home;
        this.userId = userId;
        this.name = name;
    }

    public String getAccess() {return this.access;}

    public void setAccess(String access) {this.access = access;}

    public String getHome() {return this.home;}

    public void setHome(String home) {this.home = home;}

    public String getUserId() {return this.userId;}

    public void setUserId(String userId) {this.userId = userId;}

    public String getName(){return this.name;}
}

File: app/src/main/java/com/wozart/aura/entity/model/sharing/SharedAccess.kt
package com.wozart.aura.entity.model.sharing

/***************************************************************************
 * File Name :
 * Author : Aarth Tandel
 * Date of Creation : 04/05/18
 * Description :
 * Revision History :
 * ____________________________________________________________________________
 * 29/12/17  Aarth Tandel - Initial Commit
 * ____________________________________________________________________________
 * 29/12/17 Version 1.0
 * ____________________________________________________________________________
 *
 *****************************************************************************/

class SharedAccess {
    var access = "false"
    var home: String? = null
    var userId: String? = null
    var name: String? = null

     constructor(access: String, home: String, userId: String, name: String) {
        this.access = access
        this.home = home
        this.userId = userId
        this.name = name
    }
}